commands_base/__init__.py

```diff
@@ -9,7 +9,9 @@
 import verboselogs
 
 
-def main(project_name: str):
+def main():
+    verboselogs.install()
```

Comment:
why move this here and move project_name out of the arguments?  It seems like that's making main() less useful as a function that can be called in any other context (for instance, tests or debugging).

Reply:
For context, I had the `verboselogs.install()` outside of `main()` to intentionally indicate that it needed to be the very first thing to happen and wasn't really part of the main functionality, just application boilerplate.

The environment variable lookup was just to make `main()` more functional and clear about it's expectations. I don't feel too strongly about that one.

Reply:
I see --- it's so we can call this from __main__.  In that case, though, could we just remove the `if __name__ == '__main__':` block from the bottom of this file, and *only* call it from __main__, and move the project_name lookup (and verboselogs.install() into there?

Reply:
I moved those two statements to the module level, lmk if that sounds ok with y'all

Reply:
OH.

... apparently I can't read.  But thanks :)